config/pitest-suppressions/pitest-coding-2-suppressions.xml

```diff
@@ -18,43 +18,15 @@
     <lineContent>prevScopeUninitializedVariables.pop();</lineContent>
   </mutation>
 
-
-
-
-
-
-
-
-
-
-
-  <mutation unstable="false">
-    <sourceFile>MatchXpathCheck.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.MatchXpathCheck</mutatedClass>
-    <mutatedMethod>&lt;init&gt;</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
-    <description>Removed assignment to member variable query</description>
-    <lineContent>private String query = &quot;&quot;;</lineContent>
-  </mutation>
-
   <mutation unstable="false">
     <sourceFile>MatchXpathCheck.java</sourceFile>
     <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.MatchXpathCheck</mutatedClass>
     <mutatedMethod>setQuery</mutatedMethod>
     <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
-    <description>Removed assignment to member variable query</description>
-    <lineContent>this.query = query;</lineContent>
+    <description>Removed assignment to member variable xpathExpression</description>
+    <lineContent>this.xpathExpression = null;</lineContent>
```

Comment:
This value is already by default https://pitest.org/quickstart/mutators/#EXPERIMENTAL_MEMBER_VARIABLE
and may be this will remove after https://github.com/hcoles/pitest/issues/1230